diktat-rules/src/test/resources/test/paragraph3/long_numbers/LongNumericalValuesTest.kt

```diff
@@ -0,0 +1,11 @@
+package test.paragraph3.long_numbers
+
+fun foo() {
+    val oneMillion = 1000000
+    val ten = 10
+    val creditCardNumber = 1234567890123456L
+    val socialSecurityNumber = 999999999L
+    val hexBytes = 0xFFECDE5E
+    val bytes = 0b11010010011010011001010010010010
+    val flo = 192.312341341344355345
```

Comment:
Please add test on fixer when
* real part of a float number is too long
* fractional part's length is not a multiple of allowed block length.
* float literal ends with `f`

Reply:
Added!